### PR TITLE
Adds more old nanotrasen posters

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Adds more posters to the old nanotrasen poster line
image

## Why It's Good For The Game
[soul](https://en.wikipedia.org/wiki/Soul_(2020_film))

## Changelog
:cl:
add: Adds even more nanotrasen posters~
tweak: The retro laser gun's poster has been changed to reflect the lore changes
/:cl:

### CODE REVIEW

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -695,14 +695,38 @@
...
+	desc = "A poster advertising a teleportation alternative to bluespace. It was notable for not requiring massive machinery to use, and can even work in a handheld PDA. It was shelved due to high costs and low sales. It has a nice aesthetic."
+	icon = 'whitesands/icons/obj/contraband.dmi'
+	icon_state = "poster-tc-retro"
+
+/obj/structure/sign/poster/retro/telecrystal
+	name = "Radio"
+	desc = "A poster advertising one of Nanotrasen's earliest products, a radio. One of its main selling points was a integrated OS, and dual langauge support for Galactic Common, and Draconic, which made it a smash hit. This thing is ancient."
+	icon = 'whitesands/icons/obj/contraband.dmi'
+	icon_state = "poster-tc-retro"
+
```
Comment (MarkSuckerberg) [inline code/game/objects/effects/contraband.dm]: ?

typepaths are identical, two seem to be the same

Comment (triplezeta) [reply code/game/objects/effects/contraband.dm]: also the icon_state path on the last one is wrong

Comment (rye-rice) [reply code/game/objects/effects/contraband.dm]: fixed

File: code/game/objects/effects/contraband.dm [hunk 2]
```diff
@@ -602,7 +602,7 @@
 	name = "Enlist" // but I thought deathsquad was never acknowledged
 	desc = "Enlist in the Nanotrasen Deathsquadron reserves today!"
 	icon_state = "poster29_legit" */ // I refuse to acknowledge this bullshit - Zeta
-
+// :trold:
```
Comment (ZephyrTFA) [inline code/game/objects/effects/contraband.dm]: No

**Pull request merged**